int32[6] sonar_data
int32 sec
int32 nsec
